ext/solaris/smf/puppet.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0"?>
+<!DOCTYPE service_bundle SYSTEM "/usr/share/lib/xml/dtd/service_bundle.dtd.1">
+<!-- Original puppet manifest: Luke Kanies - puppetlabs.com -->
```

Comment:
Is this the actual same one from Lak? I feel like other should get to put their names in here if he does.